crates/router/src/utils/user/password.rs

```diff
@@ -38,3 +39,20 @@ pub fn is_correct_password(
     }
     .change_context(UserErrors::InternalServerError)
 }
+
+pub fn get_temp_password() -> String {
+    let uuid_pass = uuid::Uuid::new_v4().to_string();
+    let mut rng = rand::thread_rng();
+
+    let special_chars: Vec<char> = "!@#$%^&*()-_=+[]{}|;:,.<>?".chars().collect();
+    let specialchars: char = *special_chars.choose(&mut rng).unwrap_or(&'@');
```

Comment:
variable name `special_char`

Reply:
Changed to `specialchars` to special_char